subt/ros/robot/launch/freyja.launch

```diff
@@ -174,9 +174,11 @@
   <node pkg="topic_tools" type="relay" name="points_merge_front" args="$(arg robot_name)/rgbd_front/points $(arg robot_name)/rgbd_all/points"/>
   <node pkg="topic_tools" type="relay" name="points_merge_rear" args="$(arg robot_name)/rgbd_rear/points $(arg robot_name)/rgbd_all/points"/>
 
-  <node if="0" ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
+  <node if="$(eval arg('robot_name').endswith('XM'))" ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
     <remap from="cloud_in" to="/$(arg robot_name)/rgbd_all/points"/>
     <param name="frame_id" value="odom"/>
+    <!-- resolution in meters per pixel -->
+    <param name="resolution" value="0.5" />
```

Comment:
To make sure this is intentional: We really want 0.5m x 0.5m x 0.5m voxels and not, for example, 0.05m x 0.05m x 0.05m?

Reply:
yes, it is intentional to have only rough global map for overview - 1m should be hole for the robot, so 0.5m could map it??